internal/testsuite/app.go

```diff
@@ -188,7 +188,10 @@ func (a *App) RunTests(ctx context.Context, testSpecs []*api.TestSpec) (*TestSui
 			eventLogger:          eventLogger,
 		}
 		go func() {
-			testRunner.Run(ctx)
+			err := testRunner.Run(ctx)
+			if err != nil {
+				fmt.Fprintf(a.Out, "Error with running %s", err)
```

Comment:
In this case, I think this would be correct.  If we get an error, we can output it to the test file.